pkg/openapiclient/local_crds_test.go

```diff
@@ -111,43 +95,16 @@ func Test_localCRDsClient_Paths(t *testing.T) {
 			),
 		},
 	}, {
-		name: "only fs",
-		fs:   os.DirFS("../../testcases/crds"),
+		name:        "invalid fs",
+		fileSystems: []fs.FS{os.DirFS("../../invalid")},
+		wantErr:     true,
 	}, {
-		name: "both fs and dir",
-		fs:   os.DirFS("../../testcases"),
-		dirs: []string{"crds"},
-		want: map[string]sets.Set[string]{
-			"apis/batch.x-k8s.io/v1alpha1": sets.New(
-				"batch.x-k8s.io/v1alpha1.JobSet",
-			),
-			"apis/stable.example.com/v1": sets.New(
-				"stable.example.com/v1.CELBasic",
-			),
-			"apis/acme.cert-manager.io/v1": sets.New(
-				"acme.cert-manager.io/v1.Challenge",
-				"acme.cert-manager.io/v1.Order",
-			),
-			"apis/cert-manager.io/v1": sets.New(
-				"cert-manager.io/v1.Certificate",
-				"cert-manager.io/v1.CertificateRequest",
-				"cert-manager.io/v1.ClusterIssuer",
-				"cert-manager.io/v1.Issuer",
-			),
-		},
-	}, {
-		name:    "invalid dir",
-		dirs:    []string{"invalid"},
-		wantErr: true,
-	}, {
-		name:    "invalid fs",
-		fs:      os.DirFS("../../invalid"),
-		dirs:    []string{"."},
-		wantErr: true,
-	}}
+		name:        "not a directory",
```

Comment:
This looks idential to the `invalid fs` case - is it intended to excercise something different